Dockerfile

```diff
@@ -0,0 +1,18 @@
+FROM node:lts
+
+RUN apt-get clean all
+RUN apt-get update
+RUN apt update
```

Comment:
i ran hadolint on this and got a couple items of feedback.

- `DL3009 info: Delete the apt-get lists after installing something`
- `DL3027 warning: Do not use apt as it is meant to be a end-user tool, use apt-get or apt-cache instead`

i think if you update this to:
```sh
RUN apt-get update && \
    apt-get clean && \
    rm -rf /var/lib/apt/lists/*
```

should be good

Reply:
Done!